packages/jest/src/jest.ts

```diff
@@ -5,6 +5,12 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import * as cli from 'jest-cli';
+export {
```

Comment:
followup from last question, should we ditch these exports as well?

A reason _not_ to is that we don't really have a programmatic API yet (and won't from jest 25), so all we'd do is ask people to import directly from `@jest/core` or `jest-cli`. I think that if we first ask people to change their imports, we should also give them a real API rather than the internal ones that are used now

Reply:
As long as we don't have clear vision of the new API, I wouldn't change that just yet